tensorflow/tools/pip_package/setup.py

```diff
@@ -95,6 +95,18 @@ def standard_or_nightly(standard, nightly):
     'libclang >= 13.0.0',
     # TODO(b/263178356): numpy 1.24 breaks TF's tests
     'numpy >= 1.22, <1.24',
+    # TODO(nluehr): set nvidia-* versions based on build components.
```

Comment:
@perfinion also noted this. It's a notable TODO but doesn't need to be resolved before an initial merged prototype.

Reply:
Yea, I think this needs some more thought. Will file a follow-up PR.